clustered/common/src/main/java/org/ehcache/clustered/common/internal/store/Util.java

```diff
@@ -149,23 +150,27 @@ public ByteBuffer getPayload() {
     };
   }
 
-  public static Object unmarshall(byte[] payload) {
+  public static Object unmarshallWhiteListed(byte[] payload, List<Class> whiteListedClasses) {
     ObjectInputStream objectInputStream = null;
     try {
-      objectInputStream = new ObjectInputStream(new ByteArrayInputStream(payload));
+      objectInputStream = new WhiteListObjectInputStream(new ByteArrayInputStream(payload), whiteListedClasses);
       return objectInputStream.readObject();
+    } catch (InvalidClassException e) {
```

Comment:
`InvalidClassException` is a subclass of `IOException` - we can probably eliminate the extra catch block.

Reply:
done